### DISCUSSION TITLE
Adding a graphic editior

### DISCUSSION THREAD

**Discussion by oricc** [nathanhoad/godot_dialogue_manager#168] [Ideas]
Hi!
First, thanks for a great addon!
For use by non-programmers, I was thinking of an option to add graphical editor using Godot's GraphEdit, that creates the dialogue script behind the scenes.
@nathanhoad if it seems like a good idea to you I would be happy to develop a small POC, what do you think?

**Comment by nathanhoad** [comment]
Thanks for the suggestion but I specifically built Dialogue Manager to not be graph based. I do encourage you to build out your own system if that's something you want to do though. Building an editor addon is definitely a valuable learning experience. 

If you are looking for a GUI based dialogue addon then you should give [Dialogic](https://github.com/coppolaemilio/dialogic) a go.
